.github/workflows/ci.yml

```diff
@@ -399,7 +399,7 @@ jobs:
         run: cd docs/docusaurus && yarn install && yarn test
 
   cloud-tests:
-    needs: [unit-tests, static-analysis, check-actor-permissions]
+    # needs: [unit-tests, static-analysis, check-actor-permissions]
```

Comment:
This should be reverted before merge. It is here for quicker debug cycles for this PR.